src/mcp_server/tools/observability_tools.py

```diff
@@ -0,0 +1,103 @@
+"""Observability tools for OpenShift AI monitoring and analysis.
+
+This module provides MCP tools for interacting with OpenShift AI observability data:
+- list_models: Get available AI models
+- list_namespaces: List monitored namespaces
+"""
+
+import json
+import os
+from typing import Dict, Any, List, Optional
+
+# Import core observability services
+from core.metrics import get_models_helper, get_namespaces_helper
+from core.config import PROMETHEUS_URL, THANOS_TOKEN, VERIFY_SSL
+import requests
+from datetime import datetime
+
+
+def _resp(content: str, is_error: bool = False) -> List[Dict[str, Any]]:
+    """Helper to format MCP tool responses consistently."""
+    return [{"type": "text", "text": content}]
+
+
+def list_models() -> List[Dict[str, Any]]:
+    """List all available AI models for analysis.
+    
+    Returns information about both local and external AI models available
+    for generating observability analysis and summaries.
+    
+    Returns:
+        List of available models with their configurations
+    """
+    try:
+        # Use the same logic as the metrics API
+        models = get_models_helper()
```

Comment:
Same request as above - rename it to get_models to better reflect what it's doing.